### ISSUE TITLE
make it work offline too

### ISSUE THREAD

**Issue by digitalica** [digitalica/skydivekompasroosweb#13]
it should be possible to allow the site to work offline as well. this would help using it 'in the field'. We should look into this, but only after it has been stabilized more and maybe after issue #12 (parsing canopy database at compile time)

**Comment by digitalica** [comment]
done ;-) thanks eimert

**Comment by digitalica** [comment]
Tessa Vos will build new version, no longer relevant.
